### ISSUE TITLE
low-contrast StatusLine and StatusLineNC in darker variants

### ISSUE THREAD

**Issue by tertium** [AlessandroYorba/Despacio#5]
In both Twilight and Midnight, the status line is nigh indiscernible from the normal background, which is stressful when working with several windows at once, especially with horizontal splits. I understand that Despacio generally aims for low contrast, but I'd argue that it is rather important to have visible separation between the text areas and service areas (status, splits, tabs, line numbers, etc.).

While waiting for the Pitch mode, I find it most comfortable to use Day combined with "hi Normal ctermbg=NONE" (and my terminal background is simply black).

**Comment by AlessandroYorba** [comment]
Good point! I will be introducing a new mode that will remove UI colors in despacio only leaving the text. Something similar to "hi Normal ctermbg=NONE". 

Also, I will take a new look at the statusline colors. I do see your point about the colors in the statusline. The desktop monitor I'm using was calibrated and I can see the color separation clearly but when I use MacbookPro it is very hard to tell the difference. I'll keep you posted. Here's a screen of Midnight Mode right now: [Midnight](https://cloud.githubusercontent.com/assets/11221489/19623559/5955b192-9882-11e6-9222-8a5181834ca1.png)

**Comment by tertium** [comment]
For the sake of experience, please try Midnight or Pitch with a horizontal split between two windows with some highlighting in them (say, two source code files). I definitely have a hard time seeing where ends one window and starts another one :)

**Comment by AlessandroYorba** [comment]
Agreed! The StatusLine and StatusLineNc needs to be either a lighter grey background or a stronger color.  I'll send some screenshots over later today

**Comment by AlessandroYorba** [comment]
@tertium is the StatusLine and StatusLineNC noticeable in Sunset and Twilight Mode? Is sounds to me like the 2 mode to really focus on is Midnight and Pitch

**Comment by AlessandroYorba** [comment]
The following use 
```VimL
StatusLine ctermfg=187 ctermbg=235
StatusLineNC ctermfg=240 ctermbg=235
```
<img width="1920" alt="screen shot 2016-10-26 at 7 33 20 pm" src="https://cloud.githubusercontent.com/assets/11221489/19752586/7176dad8-9bb3-11e6-8e4a-f3077502b94e.png">
<img width="1920" alt="screen shot 2016-10-26 at 7 33 54 pm" src="https://cloud.githubusercontent.com/assets/11221489/19752585/7175ff96-9bb3-11e6-9b00-3bbb5dcaaaf6.png">

**Comment by AlessandroYorba** [comment]
![screen shot 2016-10-27 at 1 02 13 am](https://cloud.githubusercontent.com/assets/11221489/19759328/5d4d9762-9be1-11e6-9106-dfe9a96ad354.png)

**Comment by AlessandroYorba** [comment]
```VimL
hi StatusLine ctermbg=137
hi StatusLineNc ctermfg=137 term=underline
```
![screen shot 2016-10-27 at 1 12 28 am](https://cloud.githubusercontent.com/assets/11221489/19759548/79b6e524-9be2-11e6-9565-f0a1eeaa26fe.png)

**Comment by AlessandroYorba** [comment]
```VimL
hi StatusLine ctermbg=179
hi StatusLineNC ctermfg=179 term=underline
```
![screen shot 2016-10-27 at 1 20 13 am](https://cloud.githubusercontent.com/assets/11221489/19759833/8c986568-9be3-11e6-9fff-3846268d7021.png)

**Comment by AlessandroYorba** [comment]
```VimL
"Midnight Mode
hi StatusLine ctermbg=66
hi StatusLineNC ctermfg=66 term=underline
```
![screen shot 2016-10-27 at 1 24 00 am](https://cloud.githubusercontent.com/assets/11221489/19759946/12848b34-9be4-11e6-9878-8f5bd39acf7b.png)

**Comment by AlessandroYorba** [comment]
```VimL
colorscheme Despacio
```
![screen shot 2016-10-27 at 3 40 11 pm](https://cloud.githubusercontent.com/assets/11221489/19788249/18b2ff54-9c5c-11e6-8979-1de6ce491d72.png)

**Comment by AlessandroYorba** [comment]
```VimL
let g:despacio_Sunset
colorscheme despacio
```
![screen shot 2016-10-27 at 3 40 54 pm](https://cloud.githubusercontent.com/assets/11221489/19788298/4ae4450a-9c5c-11e6-950c-666af7e4b8fb.png)

**Comment by AlessandroYorba** [comment]
Finally settled on ctermbg=180 for StatusLine and StatusLineNC

![screen shot 2016-10-27 at 3 41 30 pm](https://cloud.githubusercontent.com/assets/11221489/19788317/6af94200-9c5c-11e6-995c-dff99a188e51.png)

**Comment by AlessandroYorba** [comment]
```VimL
let g:despacio_Midnight = 1
colorscheme despacio
```
![screen shot 2016-10-27 at 3 42 07 pm](https://cloud.githubusercontent.com/assets/11221489/19788332/880cdc08-9c5c-11e6-9825-b50a4a6fb1ac.png)

**Comment by AlessandroYorba** [comment]
```VimL
let g:despacio_Pitch = 1
colorscheme despacio
```
![screen shot 2016-10-27 at 3 42 39 pm](https://cloud.githubusercontent.com/assets/11221489/19788358/a5a608fc-9c5c-11e6-9025-811e4b9e01aa.png)

**Comment by AlessandroYorba** [comment]
Current StatusLine is Cream color  cterm:180  hex:#dfaf87  source:http://www.colorhexa.com/dfaf87
<img width="146" alt="screen shot 2016-10-28 at 2 31 10 pm" src="https://cloud.githubusercontent.com/assets/11221489/19823029/4ad4179c-9d1b-11e6-9452-2793e5c336b9.png">

I'm considering changing to Oak color  cterm:137  hex:#af875f   http://www.colorhexa.com/af875f
<img width="147" alt="screen shot 2016-10-28 at 2 31 35 pm" src="https://cloud.githubusercontent.com/assets/11221489/19823058/838d6818-9d1b-11e6-8e31-09810d75cd16.png">

**Comment by tertium** [comment]
I spent some time pondering this, and I must admit that I do prefer your initial approach of using the same gray background for StatusLine, StatusLineNC, the window/tab delimiters, for Pmenu, etc. The current revision loses to that, in my opinion, in four regards:
- the Oak background for the active status line seems to be too bright ;
- the wild menu — which uses StatusLine for the non-selected choices — is now different from Pmenu, and Lake-next-to-Oak is not as nice as Lake-next-to-Sunset ;
- for horizontal splits in darker modes, a single-pixel Oak underline is still not visible enough ;
- for smaller fonts, underlining hinders readability.

It goes without saying that all of the above is subjective and nobody reads the passive status line anyway. Nevertheless, I would like to suggest an alternative solution: what if Despacio goes back to using a common gray background for all delimiters (including the status line), but for Midnight and Pitch (and maybe for Twighlight) this background is _brighter_ than the normal text background — not by much, but enough to make the delimiters visible, which would address the initial issue I had with them.

Just for reference, I am currently using a ctermbg=0 for Normal (pitchier than Pitch), ctermbg=235 for all delimiters including the status line (as in the Day mode before 53c361e1c41effb1dcc1f2fabe368ec06a1a5dc8), and ctermfg=244 for the text in delimiters and also for the comments (slightly brighter than in Day without blending with the normal foreground). Couldn't be happier :)

**Comment by AlessandroYorba** [comment]
@tertium good feedback for sure!!! Thank you. 
I'll post example code and screenshots for us to debate. 

BTW, I did have a very similar issue with regards to the Pmenu in [Sierra](https://github.com/AlessandroYorba/Sierra). I originally had a Pmenu that worked like version 1 of Despacio. It was one tick darker than the background for each mode set. For example, a Day( 236 ) background for Normal had a Sunset( 235 ) background for the Pmenu. Stepping down the color worked but as soon as I started using Midnight or Pitch it became increasingly harder to distinguish the Pmenu background. So I  very much like your idea of changing it to something brighter than the normal background of the mode set. This was exactly how I addressed the issue with Sierra. I used Stone ( 245 ) for the background on Pmenu for all modes and then used the mode color for the foreground. 

``` VimL
hi Pmenu ctermbg=245 ctermfg=233
```

BTW for all your custom settings are you also setting the CursorLine ctermbg=NONE? Or do you use a cursorline? If so, what color do you use?

**Comment by tertium** [comment]
I do not use CursorLine, so I didn't touch it. I tried to experiment with it now, and I agree with setting it one point brighter than the normal background color — at least for Midnight and Pitch. For Day, Sunset, and Twilight, it might be interesting to set it one point darker than the normal bg [*]. Also, I believe, CursorColumn must have the same setting as CursorLine. And ColorColumn must be something else, as it is used for different purposes.

[*] And before you reply "But one point darker would be the background of delimiters!" — I experimented with that, too, and it seems to me that I enjoy it most when for Day/Sunset/Twilight the delimiter bg is 3 points darker than the normal bg (233 for Day, 232 for Sunset, 16 for Twilight), and for Midnight/Pitch/Black it is 3 points brighter (236 for Midnight, 235 for Pitch, and 234 for Black). As for the delimiter fg and comments, 244 seems to be working okay with everything, except maybe Day (comments should be brighter) and Midnight (delimiter fg should be brighter).
